### ISSUE TITLE
Docker warning about being a link to nerdctl shows up at startup and then disappears

### ISSUE THREAD

**Issue by ericpromislow** [rancher-sandbox/rancher-desktop#663]
This happened because the docker link was added and merged into main.

Then the symlink branch stopped squelching file-system watcher changes to the utilities in
/usr/local/bin, but the warning was emitted only once, at startup. As long as changes to
/usr/local/bin/docker caused by pressing the checkbox were going to be ignored, the warning
would never be cleared.
